packages/odata-service-inquirer/src/prompts/datasources/service-url/validators.ts

```diff
@@ -0,0 +1,85 @@
+import { createForAbap, type AxiosRequestConfig, type ODataService, type ODataVersion } from '@sap-ux/axios-extension';
+import type { OdataVersion } from '@sap-ux/odata-service-writer';
```

Comment:
There is an OdataVersion import in line 1 ?

Reply:
They are different types, unfortunately!